analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/Extensions/IOperationExtensions.cs

```diff
@@ -30,6 +30,7 @@ internal static class IOperationExtensions
                 OperationKindEx.LocalReference => ILocalReferenceOperationWrapper.FromOperation(operation).Local,
                 OperationKindEx.ParameterReference => IParameterReferenceOperationWrapper.FromOperation(operation).Parameter,
                 OperationKindEx.Argument => IArgumentOperationWrapper.FromOperation(operation).Value.TrackedSymbol(),
+                OperationKindEx.DeclarationExpression => IDeclarationExpressionOperationWrapper.FromOperation(operation).Expression.TrackedSymbol(),
```

Comment:
What's this for? If I delete it, the S2222 UTs are green.

It doesn't look intentional to me.
```suggestion
```

Reply:
I added a test case for this in 8388b5b9179cb6beeb39a93427b8a5dbe88dcb79. The test cases also document how `wasCreated` and `initiallyOwned` work together.